pkgs/by-name/de/defaultbrowser/package.nix

```
@@ -3,16 +3,15 @@
   stdenv,
   fetchFromGitHub,
 }:
-
 stdenv.mkDerivation {
   pname = "defaultbrowser";
-  version = "unstable-2020-07-23";
+  version = "unstable-2024-09-04";
```

while you're changing this anyway, let's make it fit the [current conventions](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#versioning)
```suggestion
  version = "1.1-unstable-2024-09-04";
```
`1.1` is based on the [latest tag](https://github.com/kerma/defaultbrowser/releases/tag/1.1)